src/app/hooks/useOptimizelyVariation/useServerSide/index.ts

```diff
@@ -0,0 +1,30 @@
+import { MvtExperiment } from '#app/models/types/global';
+import { OptimizelyContext } from '@optimizely/react-sdk';
+import { useContext } from 'react';
+import activateExperiment from '../activateExperiment';
+
+type Props = {
+  serverSideExperiments: MvtExperiment[];
+  flagKey: string;
+};
+
+export default ({ serverSideExperiments, flagKey }: Props) => {
+  const { optimizely } = useContext(OptimizelyContext);
+
+  if (!optimizely) return null;
+
+  const experiment = serverSideExperiments.find(
+    ({ experimentName }) => experimentName === flagKey,
+  );
+
+  if (!experiment) return null;
+
+  const isEnabled = experiment.enabled;
+  const variation = isEnabled && experiment.variation;
+
+  if (!variation || variation === 'false') return null;
```

Comment:
Why don't we use destructuring here: 

```suggestion
  const { enabled, variant } = experiment;

  if (!enabled || variation === 'false') return null;
```